channeldb/invoices.go

```diff
@@ -594,7 +594,11 @@ func (d *DB) FetchAllInvoicesWithPaymentHash(pendingOnly bool) (
 
 		invoiceIndex := invoices.Bucket(invoiceIndexBucket)
 		if invoiceIndex == nil {
-			return ErrNoInvoicesCreated
+			// Mask the error if there's an invoice
+			// index as that simply means there are no
+			// invoices added yet to the DB. In this case
+			// we simply return an empty list.
+			return nil
```

Comment:
Ah, this needs to go to the earlier commit. Maybe you can group the addition of `FetchAllInvoicesWithPaymentHash` itself with this test.

Reply:
Sorry, forgot to move this the the earlier one. Fixed.